tests/browserstack_automation/page_objects/ready.page.js

```diff
@@ -24,7 +24,9 @@ class ReadyPage extends Page {
   }
 
   get ballotAddress () {
-    return $('#ballotTitleBallotAddress');
+    //return $('#ballotTitleBallotAddress');
+    return $('//span[@class ="u-cursor--pointer u-link-color u-link-underline-on-hover"]');
```

Comment:
@shan14T For the locators that dont have id's could you add them in

Reply:
Hi Francisco, for adding id's i will open a new jira issue. thanks